serverlessworkflow/modules/ROOT/pages/cloud/deploying-sonataflow-management-console-on-kubernetes.adoc

```diff
@@ -1,28 +1,26 @@
-= Deploying {product_name} Management Console on Kubernetes
+= Deploying {product_name} Management Console on {platform}
 :compat-mode!:
 // Metadata:
 :description: Deploying Management Console on Kubernetes
 :keywords: kogito, workflow, quarkus, serverless, kn, oc, openshift, sonataflow
 :table-caption: Data Set
 // envs for common content
 :management_console: SonataFlow Management Console
-:registry: OpenShift's
-:command_line_tool: oc
-:command_line_tool_name: OpenShift CLI
-:ocp_name: OpenShift Container Platform
-:platform: OpenShift
+:management_console_prefix: sonataflow-management-console
+:command_line_tool: kubectl
+:platform: Kubernetes
```

Comment:
What's the purpose of this variable?

Reply:
To make it easy to change in multiple places. 
This is my first guide here and I took inspiration from other adocs like the one below
https://github.com/apache/incubator-kie-kogito-docs/blob/b944e5cfd53346b4db0d9e3ae0e9b46fa81a4e9f/serverlessworkflow/modules/ROOT/pages/use-cases/advanced-developer-use-cases/deployments/deploying-on-kubernetes.adoc?plain=1#L19

Reply:
Yeah, I think this variable is unnecessary in this context since the document pretty much works for both K8s and OCP. We have a specialized section explaining the difference between the routes in OCP.